src/test/counterComments.test.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import InvolvementService from '../modules/InvolvementService.js';
+
+describe('Testing comments counter', () => {
```

Comment:
- [ ] Please create a new test for the implemented counter. I'm sure you will do your best.